tests/customer-resource-custom-coverage-test.js

```diff
@@ -1,6 +1,6 @@
-/* global describe, beforeEach */
+import { beforeEach, describe, it } from '@bigtest/mocha';
 import { expect } from 'chai';
-import it, { convergeOn } from './it-will';
+import { convergeOn } from '@bigtest/convergence';
```

Comment:
One thing to note, `convergeOn` is [_technically_ internal](https://github.com/thefrontside/bigtest/tree/master/packages/convergence#convergeon) even though we export it.

If want to enforce this, we should probably convert these to `new Convergence().once()`. We frequently do `convergeOn(...).then()` which will need to be changed to `.do(...)`. If the `.then()` returns a promise, `.do()` supports that as well (though not officially).

Reply:
Didn't quite read the whole PR before jumping the gun. 😆I was already reviewing before it got opened.

Reply:
I'll make another PR to change over to that pattern.

Reply:
https://github.com/folio-org/ui-eholdings/pull/268